dali/dfu/dfurun.cpp

```diff
@@ -1486,7 +1495,7 @@ class CDFUengine: public CInterface, implements IDFUengine
                         }
                         else if (foreigncopy||auxfdesc)
                         {
-                            IFileDescriptor * srcDesc = (auxfdesc.get() ? auxfdesc.get() : foreignfdesc.get());
+                            IFileDescriptor * srcDesc = (auxfdesc.get() ? auxfdesc.get() : srcFile->getFileDescriptor());
```

Comment:
leaks the IFileDescriptor if getFileDescriptor called here.

Reply:
Changed